src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/NodeService.kt

```diff
@@ -78,8 +83,16 @@ class NodeService(
                 this.transactionId = transactionId.toString()
                 totalAmount = (ev.data.amount.plus(ev.data.fee)).toBigDecimal()
                 fee = ev.data.fee.toBigDecimal()
-                timestampOperation = OffsetDateTime.now()
-                additionalPaymentInformations = mapOf()
+                timestampOperation = timestamp
+                additionalPaymentInformations =
+                  mapOf(
+                    "outcome_payment_gateway" to transactionOutcome.value,
+                    "authorization_code" to authorizationCode.orElseGet { "" },
```

Comment:
Is it correct to write an empty string?

Reply:
No, but consider 3 factors:

1. this case is not real. If we have an outcome ok we should have also info about the authorization code
2. this fields are provided as info to the nodo, but it doesn't make any elaboration of this
3. this call will be refactored in the next sprint, since we need to remove synchro call in transaction service and use the queue implemented here